editor/src/components/canvas/canvas-strategies/absolute-move-strategy.spec.browser2.tsx

```diff
@@ -202,6 +202,10 @@ export var ${BakedInStoryboardVariableName} = (props) => {
 `
 }
 
+function positioningFromCss({ left, top }: CSSStyleDeclaration) {
```

Comment:
super minor nitpick, but I _think_ we generally avoid object destructuring in function parameters. I'm not sure if it's a rule, or we only avoid when in combination with type annotations (because that makes it look silly)

Reply:
I also think it looks a bit strange but at the same time it's somewhat useful in the case of these selector type functions

Reply:
refactoring

Reply:
TBH I like them, because they express in the signature which part of the parameter is really used in the function body.